MekHQ/src/mekhq/campaign/rating/CampaignOpsReputation.java

```diff
@@ -323,6 +323,8 @@ private void calcNeededTechs() {
 
     private void updatePersonnelCounts() {
         setNonAdminPersonnelCount(0);
+        technicians = 0;
```

Comment:
This should probably be added to the initValues() logic as well.

Reply:
initValues() calls updatePersonnelCounts().

Reply:
You're right.  Forgot about that.